frontend_app_admin_workspace_user/src/component/AddUserForm.jsx

```diff
@@ -115,7 +115,7 @@ export class AddUserForm extends React.Component {
             data-cy='adduser_username'
           />
 
-          {!props.newUsernameAvailability && (
+          {!props.newUsernameAvailability && state.newUserUsername !== '' && (
```

Comment:
Shouldn't we name this `state.newUsername`?

Reply:
Since the new username is not linked to the user making the change but to the new user being created, I think I prefer the pattern newUserXXX.